charts/pathfinder-rpc-server/Chart.yaml

```
@@ -0,0 +1,12 @@
+name: pathfinder-rpc-server
+description: pathfinder-rpc-server helm chart 
+version: 0.1.0
```

We could try to use Semantic versioning, and mark this one as "stable" by using an even number.

```suggestion
version: 0.2.0
```